test/e2e/plain-v0_provisioner_test.go

```diff
@@ -192,3 +233,15 @@ var _ = Describe("plain-v0 provisioner", func() {
 		})
 	})
 })
+
+func newBundleLabelSelector(bundle *v1alpha1.Bundle) labels.Selector {
```

Comment:
Follow-up/TODO: Maybe an internal util we can reuse?

Reply:
Sounds reasonable - moved this to the internal package.